benchmark/buffers/buffer-iterate.js

```diff
@@ -29,20 +29,23 @@ function main(conf) {
 function benchFor(buffer, n) {
   bench.start();
 
-  for (var k = 0; k < n; k++)
-    for (var i = 0; i < buffer.length; i++)
+  for (var k = 0; k < n; k++) {
```

Comment:
I get why braces would be added here, but why on the `for` below if it does not contain a multi-line body?